build.rs

```diff
@@ -2,20 +2,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#![feature(os,old_io)]
-
-use std::old_io::process::{Command, ProcessExit, StdioContainer};
-use std::os;
-
+use std::env;
+use std::process::{Command, Stdio};
 
 fn main() {
-    let out_dir = os::getenv("OUT_DIR").unwrap();
+    let out_dir = env::var("OUT_DIR").unwrap();
     let result = Command::new("make")
         .args(&["-f", "makefile.cargo"])
-        .stdout(StdioContainer::InheritFd(1))
-        .stderr(StdioContainer::InheritFd(2))
+        .stdout(Stdio::inherit())
+        .stderr(Stdio::inherit())
```

Comment:
`Stdio::inherit()` is the default for `Command::status`, so these two lines can be removed. r=me with this.